apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -48,6 +49,10 @@ type DatadogFeatures struct {
 	ProcessCollection *ProcessCollectionFeatureConfig `json:"processCollection,omitempty"`
 	// ContainerCollection configuration.
 	ContainerCollection *ContainerCollectionFeatureConfig `json:"containerCollection,omitempty"`
```

Comment:
Can we change this to LiveContainerCollection?

Reply:
`LiveProcessCollection` as well?

Reply:
Implemented both